add argument to include removed files in the output

yigit opened the pull request.

yigit committed:
add argument to include removed files in the output

dlam commented on find_changed_files.sh:
```diff
@@ -1,28 +1,38 @@
 cat $GITHUB_EVENT_PATH
 COMPARE=$(jq '.compare' $GITHUB_EVENT_PATH)
 PR=$(jq -r '.pull_request._links.self.href' $GITHUB_EVENT_PATH)
 echo "-----"
```
While we're here, can we also clean up these echo statements since they're redundant?

yigit replied:
which statements?

yigit committed:
add support for single commit changes

in that case, url is just a single commit so we need to use the commit api to get it

yigit commented:
added support for single commits as well. this is important as without that, build will break when we report 0 files changed.
